pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -940,6 +940,7 @@ default-package-overrides:
   - hp2pretty ==0.9
   - hpack ==0.31.2
   - hpack-dhall ==0.5.2
+  - hprotoc ==2.4.13
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/75449#discussion_r356361205.

You'll have to create overrides for `hprotoc` versions in `configuration-common.nix` (and `protocol-buffers-descriptor_2_4_13`).

Also, please don't edit the `hackage-packages.nix` file.  This is generated automatically by peti.